packages/quantic/force-app/main/default/lwc/quanticStandaloneSearchBox/__tests__/quanticStandaloneSearchBox.test.js

```diff
@@ -109,6 +119,107 @@ describe('c-quantic-standalone-search-box', () => {
   afterEach(() => {
     cleanup();
   });
+  describe('with default options', () => {
+    it('should create the component', async () => {
+      const element = createTestComponent();
+      await flushPromises();
+
+      expect(element).not.toBeNull();
+
+      const interfaceElem = element.shadowRoot.querySelector(
+        selectors.searchInterface
+      );
+      expect(interfaceElem).not.toBeNull();
+
+      const input = interfaceElem.querySelector(selectors.searchBoxInput);
+      expect(input).not.toBeNull();
+      expect(input.withoutSubmitButton).toBe(false);
+      expect(input.textarea).toBe(false);
+      expect(input.placeholder).toBe(null);
+      expect(input.inputValue).toBe('');
+      expect(input.recentQueries).toBeUndefined();
+      expect(input.maxNumberOfSuggestions).toBe(7);
+    });
+  });
+
+  describe('with custom options', () => {
+    it('should create the component with custom options', async () => {
+      const element = createTestComponent({
+        engineId: exampleEngine.id,
+        placeholder: 'place',
+        withoutSubmitButton: true,
+        numberOfSuggestions: 10,
+        keepFiltersOnSearch: true,
+        redirectUrl: '/custom-search/%40uri',
+        searchHub: 'customHub',
+        pipeline: 'customPipeline',
+        textarea: true,
+        disableRecentQueries: true,
```

Comment:
here maybe it would be cleaner to declare at the top of the file a customOptions object and iterate over them in your expect block instead of writing a bunch of expectations?